portfolio/src/main/webapp/script.js

```diff
@@ -27,13 +27,13 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-function randomXDQuote(){
```

Comment:
if you change the name of the function here, you also have to change the name where you are calling it (index.html)

Reply:
you're right thank u for catching that!